planetscale/resource_branch_password.go

```diff
@@ -0,0 +1,270 @@
+package planetscale
+
+import (
+	"context"
+	"errors"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	ps "github.com/planetscale/planetscale-go/planetscale"
+	"strings"
+)
+
+func resourceBranchPassword() *schema.Resource {
+
+	return &schema.Resource{
+		Create: resourceBranchPasswordCreate,
+		Read:   resourceBranchPasswordRead,
+		Delete: resourceBranchPasswordDelete,
+		Schema: map[string]*schema.Schema{
+			"organization": {
+				Type:        schema.TypeString,
+				Description: "organization to create database under",
+				Required:    true,
+				ForceNew:    true,
+			},
+			"database": {
+				Type:        schema.TypeString,
+				Description: "display name of your database",
+				Required:    true,
+				ForceNew:    true,
+			},
+			"branch": {
+				Type:        schema.TypeString,
+				Description: "display name of your branch",
+				Optional:    true,
+				ForceNew:    true,
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Description: "name of your password",
```

Comment:
@cemo  is this the actual value of the password or the name of the password?

Reply:
It is actually seems on UI when you create a password. Every password is named on PS.

Reply:
If you can give a try to create a password on UI, you will immediately see.

Reply:
👍🏽 Would do

Reply:
Had a chance to look? Sorry to push but we need a release :)